rootfs-builder/alpine/config.sh

```diff
@@ -16,3 +16,4 @@ MIRROR=http://dl-5.alpinelinux.org/alpine
 # Mandatory Packages that must be installed
 #  - iptables: Need by Kata agent
 PACKAGES="iptables"
+[ "$SECCOMP" == "yes" ] && PACKAGES+=" libseccomp" || true
```

Comment:
You don't actually need the `|| true` here so you can simplify to the following (note the single equals too):

```
[ "$SECCOMP" = "yes" ] && PACKAGES+=" libseccomp"
```

Reply:
@jodh-intel : Updated!

Reply:
@jodh-intel :  I guess your comments apply here as well
https://github.com/kata-containers/osbuilder/blob/b7d11db1a1898ab94eb2dcf5e43ec9fa376e5c15/rootfs-builder/centos/config.sh#L30
Another PR to optimize it?
If I am right, the purpose of adding `|| true` is to tell `set -e` in the script its ok to fail here and not error out and stop.

Reply:
That is what `|| true` is for, yes. But that test won't ever "fail" - it just won't match so `set -e` will not exit if `"$AGENT_INIT" != "no"`.

You could fix that on a separate commit on this PR if you want rather than creating a whole new PR for it ;)